awd/database.py

```diff
@@ -0,0 +1,106 @@
+from __future__ import annotations
+
+import datetime
+import logging
+from typing import Any
+
+from pynamodb.attributes import NumberAttribute, UnicodeAttribute
+from pynamodb.models import Model
+
+from awd.config import DATE_FORMAT
+from awd.status import Status
+
+logger = logging.getLogger(__name__)
+
+
+class DoiProcessAttempt(Model):
+    """A class modeling the DynamoDB table."""
+
+    class Meta:  # noqa: D106
+        table_name = "None"
```

Comment:
Totally optional, but a comment might be nice here to explain where and how the table name is set.  Even after poking around a little bit, I'm unsure myself!